include/zephyr/dt-bindings/power/power.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_POWER_POWER_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_POWER_POWER_H_
+
+#include <limits.h>
```

Comment:
this is DT, can this be used?

Reply:
that is good question, this header contains these constants that depending on type can change according with compile and runtime machine. Re-define such things is error prone, in other hand this is DT....